src/tools.ts

```diff
@@ -269,3 +269,58 @@ export async function collapseCodeSections (editor: monaco.editor.IStandaloneCod
     }
   }
 }
+
+export function mapClipboard (
+  editor: monaco.editor.IStandaloneCodeEditor, {
+    toClipboard = v => v,
+    fromClipboard = v => v
+  }: {
+    toClipboard: (v: string) => string
+    fromClipboard: (v: string) => string
+  }): void {
+  let copiedText = ''
+  editor.addCommand(monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyC, () => {
```

Comment:
Not sure why it's necessary, wouldn't it call `editor.action.clipboardCopyAction` by default ?

Reply:
It's not even required